README.md

```diff
@@ -5,23 +5,16 @@
 # OpenTelemetry RFCs
 ## Evolving OpenTelemetry at the speed of Markdown
 
-The OpenTelemetry "RFC" (request for comments) process aims to track the **rationale** behind *new functionality*, including trade-offs and alternatives that have been considered.
-
-Much like how designers may improve user research feedback cycles by using prototypes instead of production-ready implementations, we hope that RFCs will help the OpenTelemetry consider functionality more quickly and thoroughly than, say, jumping directly into the standard pull request workflow.
-
-Bug fixes and changes with no external impact, such as refactors, can skip the RFC process and instead stick with the standard GitHub tooling :)
+OpenTelemetry uses an "RFC" (request for comments) process for proposing changes to the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification).
```

Comment:
```suggestion
OpenTelemetry uses an "RFC" (request for comments) process for proposing changes to the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification).
```

Reply:
minor, up to you @tedsuo

Reply:
Yeah, I thought about that. In the spec repo itself, we say OpenTelemetry specification (no caps), so I matched that for now. 🤷‍♂